### DISCUSSION TITLE
Kustomize decryption.secretRef.name not required when using AWS KMS

### DISCUSSION THREAD

**Discussion by cer** [fluxcd/flux2#4979] [General]
I noticed something potentially confusing in the docs.

The [docs](https://fluxcd.io/flux/components/kustomize/kustomizations/#decryption) say:

```
It has two required fields:
.secretRef.name: The name of the secret that contains the keys to be used for decryption.
.provider: The secrets decryption provider to be used. The only supported value at the moment is sops.
```

However, `.secretRef.name` should not be specified when using AWS KMS.

**Comment by stefanprodan** [comment]
Thanks @cer I made a clarification here https://github.com/fluxcd/kustomize-controller/pull/1242
